packages/big-design/src/components/Modal/Modal.tsx

```diff
@@ -29,42 +31,20 @@ interface ModalState {
   modalContainer: HTMLDivElement | null;
 }
 
-export interface ModalActionsProps {
-  withBorder?: boolean;
-}
-
-export interface ModalHeaderProps {
-  withBorder?: boolean;
+interface Action extends Omit<ButtonProps, 'children'> {
```

Comment:
What do you think about renaming this `ModalAction`? We also need to export it.

Reply:
Why export it?

Reply:
For example, if you don't want to inline the `actions` prop you could do.

```tsx
const actions: ModalAction[] = [
  { text: 'Cancel', onClick: () => {} }
];

// And use it like
<Modal actions={actions} />
```